### ISSUE TITLE
Question 2.3: Conjugate forms 3

### ISSUE THREAD

**Issue by smith-garrett** [vasishth/bayescogsci#51]
I got thrown off by an attachment ambiguity in the description of the data: "Suppose that we have data $x_1$, ..., $x_n$, with sample size $n$ that is exponentially distributed." I read this as $n$ being exponentially distributed instead of the $x_i$ being exponentially distributed.

Could maybe reword to, "Suppose we have $n$ data points that are drawn from an exponential distribution: $x_1, \dots, x_n$."

**Comment by vasishth** [comment]
fixed. the text will be updated when Bruno compiles the book next.
